pkg/extension/sumologicextension/credentials.go

```diff
@@ -48,7 +48,8 @@ type credsGetter struct {
 // CheckCollectorCredentials checks if collector credentials can be found in path
 // configured in the config.
 func (cr credsGetter) CheckCollectorCredentials() bool {
-	filenameHash, err := hash(cr.conf.CollectorName)
+	key := cr.conf.CollectorName + cr.conf.Credentials.AccessID + cr.conf.Credentials.AccessKey
+	filenameHash, err := hash(key)
 	if err != nil {
```

Comment:
Sorry, this is not strictly related to this PR, but shouldn't we be logging these errors?